evidence.go

```diff
@@ -200,15 +202,26 @@ func (e *CcaEvidence) Verify(iak crypto.PublicKey, rpk crypto.PublicKey) error {
 		return fmt.Errorf("missing CCA realm Token")
 	}
 
+	// extract RAK from the realm token
+	rawRAK, err := e.CcaRealmClaims.GetPubKey()
```

Comment:
```suggestion
if e.CcaRealmClaims == nil {
    return fmt.Errorf("cannot verify realm token, missing realm claims")
}
	rawRAK, err := e.CcaRealmClaims.GetPubKey()
```

Reply:
is it possible to get here in such state?

Reply:
Yes, in case FromCBOR fails but user call Verify API subsequently! Then it is better to Fail this rather than Panic?

Reply:
exactly, if they decide to go ahead ignoring the error they are in the rough.  panic'ing is entirely acceptable in this situation.

Reply:
We want to make life simple for the user, even if they make mistake..
Anyway, I would have preferred this simple check to cleanly understand the reason for failure rather than panic in this way!

Reply:
it's not simplifying their lives in any way.  If they ignored the API contract in the first place they might as well ignore the contract in this case too.